tools/src/versioning/android/transforms/vendoredModulesTransforms.ts

```diff
@@ -24,11 +24,6 @@ export function vendoredModulesTransforms(prefix: string): Record<string, FileTr
           find: '$nodeModules/versioned-react-native/ReactAndroid/gradle.properties',
           replaceWith: '$defaultDir/gradle.properties',
         },
-        {
-          paths: 'ExponentPackage.kt',
-          find: 'import com.shopify',
-          replaceWith: `import ${prefix}.com.shopify`,
-        },
```

Comment:
unused code. the ExponentPackage.kt versioning is actually in `exponentPackageTransforms`.